src/main/java/org/minions/devfund/chelo/WhackAMole.java

```diff
@@ -0,0 +1,216 @@
+package org.minions.devfund.chelo;
+
+import java.util.Random;
+import java.util.Scanner;
+
+/**
+ * A class for WackAMole task.
+ */
+public class WhackAMole {
+    private int score;
+    private int molesLeft;
+    private int attemptsLeft;
+    private char[][] moleGrid;
+
+    /**
+     * Constructor for the game, specifies total number of whacks allowed, and the grid dimension.
+     * @param numAttempts initial tries for the user.
+     * @param gridDimension size of the grid.
+     */
+    WhackAMole(int numAttempts, int gridDimension) {
```

Comment:
Since your pull request is still open.
please write your unit tests something like this.
`
Running: testConstructorGrid
Finished: testConstructorGrid
Running: testConstructorNumOfAttempts
Finished: testConstructorNumOfAttempts
Running: testPlaceFail
Finished: testPlaceFail
Running: testWhackSucceedAttempts
Finished: testWhackSucceedAttempts
Running: testWhackFailAttempts
Finished: testWhackFailAttempts
Running: testPlaceSucceed
Finished: testPlaceSucceed
Running: testWhackFailMolesLeft
Finished: testWhackFailMolesLeft
Running: testWhackSucceedMolesLeft
Finished: testWhackSucceedMolesLeft
Tests run: 8, Passed: 8, Failed: 0
`